scene/gui/tree.cpp

```diff
@@ -4990,6 +4990,26 @@ TreeItem *Tree::get_item_with_text(const String &p_find) const {
 	return nullptr;
 }
 
+TreeItem *Tree::get_item_with_metadata(const Variant &p_find, int p_column) const {
+	if (p_column < 0) {
```

Comment:
I think this is reversed? Column greater or equal 0 is a specific one.

Reply:
🤦‍♂️ Fixed.